run-admin-web-application.sh

```diff
@@ -2,13 +2,15 @@
 
 # Script to run the CoatRack admin web application 
 
+# Optional: pass a Spring profile name as first command line argument 
```

Comment:
So regarding the extra argument with the profile name. The user might not see these comments, only if he opens it with a notepad or notepad++ for example,
Therefore, my question is should we also add this to the readme?

Reply:
In general I agree, however right now only CoatRack developers can create a custom profile, so right now it is not yet really relevant for others.

Reply:
I added it to the README.

Reply:
I checked and it looks very nice. So I consider this conversation solved